submission/treasurehunter/firefox/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "manifest_version": 2,
```

Comment:
please use Manifest Version 3

Reply:
probably because of v3, i run it in brave but will check